cryodrgn/commands/abinit_het.py

```diff
@@ -881,6 +870,17 @@ def main(args):
     else:
         start_epoch = 0
 
+    # parallelize
```

Comment:
This entire block (determination of whether to use `DataParallel` or not) has been moved from before loading the checkpoint, to after loading the checkpoint. Since the state dict that is saved is now of the underlying model, not `DataParallel` (through `unparallelize`), this wrapping should come later.

PR [190](https://github.com/zhonge/cryodrgn/pull/190) had moved this block before loading the checkpoint, which was correct, but had the limitation that `--multigpu` had to remain the same before/after loading the checkpoint.

Simply adding `unparallelize` and not touching the rest of the code was always the correct fix there.